src/extension/background-script/actions/allowances/delete.ts

```diff
@@ -3,6 +3,7 @@ import type { MessageAllowanceDelete } from "~/types";
 
 const deleteAllowance = async (message: MessageAllowanceDelete) => {
   const id = message.args.id;
+  if (!id) return;
```

Comment:
this probably should return an error like `{ error: "id is missing" }`

to make sure that all actions always return the same response format